src/editpage/components/response/Goal.vue

```diff
@@ -79,12 +84,18 @@ export default {
     },
     subGoal: function() {
       if (this.parts.length > 1) {
-        return this.parts[1].trim().replace(/\n/g, '<br>');
+        console.log(this.parts[1].trim().replace(
+            /\n/g, '<br>').replace(/<br> {2}/g, '<br>&emsp;&nbsp;'));
```

Comment:
Let's remove this console log to not spam the console, we can always add this back when debugging

Reply:
Forgot to take that off, just had it for debugging.